### DISCUSSION TITLE
前后括号大小不一致，请问该如何修改呢，

### DISCUSSION THREAD

**Discussion by presto-wei** [obster-y/XJTU-thesis#101] [General]
![image](https://user-images.githubusercontent.com/27005213/234475621-8601819c-9912-4997-a419-c4b07757351b.png)
\begin{equation}\label{eqn:c3:perceptual_loss}
  L_{\text {Perceptual}}\left(G\right)=\frac{1}{C_j H_j W_j} \left \| \varphi_j \left ( y \right )  - \varphi_j \left(G\left(x\right)\right) \right \|_2^2
\end{equation}

**Comment by obster-y** [comment]
答案：换个字体

另外：为啥要修改

**Comment by presto-wei** [comment]
那我先不修改了，如果盲审老师提意见，我再修改字体，谢谢大佬

**Comment by obster-y** [comment]
可以考虑在某些括号里不用 `\left \right`

**Comment by presto-wei** [comment]
嗯，我尝试一下，

**Comment by presto-wei** [comment]
![image](https://user-images.githubusercontent.com/27005213/234479977-445366bb-5f77-496a-b2f6-a4b19aeebc2f.png)
删除前面较大括号的left和right，间距会变小，不过我修改成这样，后续老师有意见，我再修改
